src/Common/AsynchronousMetrics.cpp

```diff
@@ -68,6 +68,8 @@ AsynchronousMetrics::AsynchronousMetrics(
 
     openFileIfExists("/sys/fs/cgroup/memory/memory.limit_in_bytes", cgroupmem_limit_in_bytes);
     openFileIfExists("/sys/fs/cgroup/memory/memory.usage_in_bytes", cgroupmem_usage_in_bytes);
+    openFileIfExists("/sys/fs/cgroup/memory.max", cgroupmem_v2_limit_in_bytes);
```

Comment:
Missing the changes in AsynchronousMetrics.h

Reply:
will fix

Reply:
done